.github/workflows/build_native_linux_packages.yml

```diff
@@ -0,0 +1,131 @@
+name: Build Native Linux Packages
+
+on:
+  workflow_call:
+    inputs:
+      amdgpu_family:
+        description: gfx arch for creating the artifact fetch url
+        required: true
+        type: string
+      artifact_run_id:
+        description: workflow run id to download the artifacts from
+        type: string
+      rocm_version:
+        description: ROCm version to append to the package( Like 8.0.0 or 8.1.0).
+        required: true
+        type: string
+      package_type:
+        description: Specify whether debian or rpm packages are needed (deb or rpm).
+        required: true
+        type: string
+      package_suffix:
+        description: The suffix to be added to package name(build_no or master or rc or combiantion).
+        required: false
+        type: string
+  workflow_dispatch:
+    inputs:
+      amdgpu_family:
+        type: choice
+        options:
+          - gfx110X-dgpu
```

Comment:
I realize you copied/pasted this, but I've been nuking these fixed lists in parameters. There's no way we will keep them up to date.